src/css/Home.css

```diff
@@ -0,0 +1,65 @@
+h1 {
```

Comment:
i thought we're using scss now?

Reply:
Yes.... SCSS is preprocessed and turned into css. That's why there are css files here.

Reply:
Good point about the `.gitignore`. I'll remove the css and add it to the `.gitignore`.